.github/workflows/cypress-workflow.yml

```diff
@@ -35,6 +35,9 @@ jobs:
           cd index-management
           ./gradlew run -Dopensearch.version=${{ env.OPENSEARCH_VERSION }} &
           sleep 300
+        # cd index-management
+        # ./gradlew run -PcustomDistributionUrl="https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.7.0/7747/linux/x64/tar/dist/opensearch/opensearch-2.7.0-linux-x64.tar.gz" &
```

Comment:
Should those new comment out lines be removed?

Reply:
Sure, and this PR should be merged after the notification PR.